watcher/Dockerfile

```diff
@@ -28,26 +41,10 @@ RUN source /opt/lsst/software/stack/loadLSST.bash && setup lsst_distrib && \
                       ScriptQueue \
                       Script \
                       ATCamera \
-                      ATSpectrograph
-
-WORKDIR /home/saluser/repos/ts_xml
-RUN git fetch --all && git checkout "tags/v4.3.0_RC1" -b "v4.3.0_RC1"
-RUN source /opt/lsst/software/stack/loadLSST.bash && setup lsst_distrib && \
-    source /home/saluser/repos/ts_sal/setup.env && \
-    setup ts_sal -t current && \
-    make_idl_files.py Watcher
-
-WORKDIR /home/saluser/repos/ts_watcher
-RUN source /opt/lsst/software/stack/loadLSST.bash && setup lsst_distrib && \
-    source /home/saluser/repos/ts_sal/setup.env && \
-    eups declare -r . ts_watcher -t current && \
-    setup ts_salobj -t current && \
-    setup ts_watcher -t current && \
-    scons
-
-WORKDIR /home/saluser/repos/ts_config_ocs
-RUN git fetch --all && \
-    git checkout "tickets/DM-21742"
+                      ATSpectrograph \
+                      ATArchiver \
+                      ATSpectrograph \
```

Comment:
Looks like ATSpectrograph is duplicated here. Is that intentional?

Reply:
Fixed! Thanks!